modules/monitoring-specifying-how-a-service-is-monitored-by-cluster-observability-operator.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// monitoring/cluster-observability-operator/configuring-the-cluster-observability-operator-to-monitor-a-service.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="specifying-how-a-service-is-monitored-by-cluster-observability-operator_{context}"]
+= Specifying how a service is monitored by Cluster Observability Operator
+
+To use the metrics exposed by the sample service you created in the "Deploying a sample service for Cluster Observability Operator" section, you must configure monitoring components to scrape metrics from the `/metrics` endpoint. 
+
+You can create this configuration by using a `ServiceMonitor` custom resource definition (CRD) that specifies how the service should be monitored, or a `PodMonitor` CRD that specifies how a pod should be monitored. 
```

Comment:
```suggestion
You can create this configuration by using a `ServiceMonitor` custom resource (CR) that specifies how the service should be monitored, or a `PodMonitor` CR that specifies how a pod should be monitored. 
```
I think someone else mentioned this during their review also, pretty sure these are CRs instead of CRDs, or you could just use "object" across the board for consistency?

Reply:
QE says [here](https://github.com/openshift/openshift-docs/pull/67608#discussion_r1395149438) that "Both `ServiceMonitor` and `PodMonitor` are CRDs, here you create CRs based these CRD"

Reply:
But I'm going to change these both to "object" to be consistent with the other sections. Thanks!